src/components/inputs/rich-text-input/dropdown.styles.js

```diff
@@ -0,0 +1,108 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import vars from '../../../../materials/custom-properties';
+
+const getItemStyles = props => {
```

Comment:
Following my previous comment about the `:hover` selector, this function can be removed and we'd have:
```
const DropdownItem = styled.div`
  padding: ${vars.spacingXs} ${vars.spacingS};
  font-size: ${vars.fontSizeForInput};
  font-family: ${vars.fontFamilyDefault};

  background-color: ${props =>
    props.isSelected ? vars.colorAccent95 : vars.colorSurface};

  &:hover {
    background-color: ${vars.colorNeutral90};
  }
`;
```